src/single-spa-react.js

```diff
@@ -217,13 +217,18 @@ function defaultDomElementGetter(props) {
 }
 
 function reactDomRender({ opts, elementToRender, domElement, whenFinished }) {
-  if (opts.renderType === "createRoot") {
-    return opts.ReactDOM.createRoot(domElement).render(
+  if ([
+    "createRoot",
+    "unstable_createRoot",
+    "createBlockingRoot",
+    "unstable_createBlockingRoot",
+  ].includes(opts.renderType)) {
+    return opts.ReactDOM[opts.renderType](domElement).render(
       elementToRender,
-      whenFinished
+      whenFinished,
```

Comment:
This is incorrect prettier formatting for our version of prettier. This is caused by the git pre-commit hook being skipped, probably due to modifying the file in the Github UI instead of in an IDE where git hooks are run automatically.

Reply:
To fix, run `yarn format` locally